### PR TITLE
LPS-139478 - There is no warning message when resetting a page that has fields being used in a rule

### PR DESCRIPTION
**Pull request by alberttgomes**
I create new file to check if fields with rules applied when page reset.
And new language to modal Page Reset.
link LPS: https://issues.liferay.com/browse/LPS-139478

### CODE REVIEW

**Comment by alberttgomes** [comment]
ci:test:sf

**Comment by alberttgomes** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -91,6 +91,7 @@ a-page-variation-with-that-name-already-exists=A page variation with that name a
 a-pingback-is-a-comment-that-is-created-when-you-link-to-another-blog-post-where-pingbacks-are-enabled=A pingback is a comment that is created when you link to another blog post where pingbacks are enabled.
 a-previous-version=A Previous Version
 a-remotely-staged-asset-library-cannot-be-connected-to-a-locally-staged-site=A remotely staged asset library cannot be connected to a locally staged site.
+a-rule-is-applied-to-fields-of-this-page=A rule is applied to fields of this page. By deleting it, it will also break the rule that is using it
```
Comment (alinedoleron) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:94]: Hi @alberttgomes. You forgot the dot in the end of the phrase. Please, could you fix it?

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -4924,6 +4925,7 @@ delete-missing-layouts-staging-help=If this option is chosen, all the pages in t
 delete-notifications=Delete Notifications
 delete-object-definition=Delete Object Definition
 delete-page=Delete Page
+delete-page-with-rule-applied-to-field=Reset Page with Rule Applied to Field
```
Comment (alinedoleron) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:4928]: Hi @alberttgomes. The language key here should be: `reset-page-with-rule-applied-to-field`

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Hey @alberttgomes! Good job!

I've requested a few changes on language key file. Remember to run again the `gw buildLang` after changing the keys.

Thanks!

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 2 above)
Comment (alberttgomes) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:4928]: Hi @alinedoleron, thank you!!! I'm making your requests!!

Comment (alberttgomes) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: hey @alinedoleron i need run the ci again?

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alberttgomes** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alberttgomes** [comment]
ci:test:sf

**Comment by alberttgomes** [comment]
ci:test:relevant

**Pull request closed without merging**